watches.yaml

```
@@ -4,3 +4,10 @@
   group: cost-mgmt.openshift.io
   kind: CostManagement
   role: /opt/ansible/roles/setup
+
+# collect the reports
+- version: v1alpha1
+  group: cost-mgmt-data.openshift.io
+  kind: CostManagementData
+  role: /opt/ansible/roles/collect
+  reconcilePeriod: 60m
```

I doubt it, but is there anyway to make this configurable?

If not we should probably back it off to every 6 hours.
```suggestion
  reconcilePeriod: 360m
```

---

So far I don't see a way to make it configurable - for now I will back it off to every 6 hours - but I will keep looking to see if I can figure out how to make it configurable/if possible.